### PR TITLE
Update deployment scripts for 2.0.4 and test campaigns in central couch

### LINKED ISSUE #11073: Run WMAgent validation for 2.0.4 production release candidate
**Impact of the new feature**
WMAgent

**Is your feature request related to a problem? Please describe.**
By-monthly task

**Describe the solution you'd like**
We need to release a new WMAgent stable version in June, so this issue is to keep track of the full validation activity to be carried out in the first weeks of June, once we have a final 2.0.4 tag.

It also includes the creation of WMAgent release notes and the validation check-list twiki.

**Describe alternatives you've considered**
none

**Additional context**
none

### PR DESCRIPTION
**Pull request by amaltaro**
Fixes #11073 

#### Status
In development

#### Description
Short summary of changes is:
* remove 2 deployment variables no longer needed (DASHBOARD_URL and REQMGR_URL)
* update deployment script examples to the latest WMAgent production release candidate
* update test campaigns to be inserted in central CouchDB
* data location must be set with the RSE name (not the site name)

But the real work in the relevant GH issue is actually the full validation of a new stable WMAgent release...

#### Is it backward compatible (if not, which system it affects?)
YEs

#### Related PRs
None

#### External dependencies / deployment changes
None

### CODE REVIEW

**Comment by amaltaro** [comment]
test this please

**Comment by amaltaro** [comment]
test this please

File: deploy/deploy-wmagent.sh
Comment (amaltaro) [inline deploy/deploy-wmagent.sh:264]: This will be fixed once the deployment PR gets merged.

**Comment by amaltaro** [comment]
@todor-ivanov this is the usual deployment changes that come with WMAgent upgrade campaigns, but just in case you want to have a look. I am still waiting for a PR to be merged in the deployment repository though.

File: deploy/WMAgent.production
Comment (todor-ivanov) [inline deploy/WMAgent.production:13]: Sorry if I have missed a change in the past that is explaining this already, but why do we need to remove this URL from the agent's configuration?

File: deploy/deploy-wmagent.sh
Comment (todor-ivanov) [inline deploy/deploy-wmagent.sh:134]: Just a comment here, No need for a change in this PR. We'll have to think of a better way of distributing this instead of hardcoding a machine name in the deployment script. But is not worth the effort before we figure out how we should proceed with the token based authentication.

File: deploy/deploy-wmagent.sh
Comment (todor-ivanov) [reply deploy/deploy-wmagent.sh:264]: @amaltaro , why do we need to have patch numbers hard coded somewhere in the deployment script at the first place, or is it just a reminder for which patch needs to be added ?

**Review by todor-ivanov — APPROVED** [review]
Thanks @amaltaro I've left some insignificant comments inline. None of them are a request for change. The PR is good.

File: deploy/WMAgent.production
Comment (amaltaro) [reply deploy/WMAgent.production:13]: It's been replaced by REQMGR2_URL (long time ago), this was a leftover.

File: deploy/deploy-wmagent.sh
Comment (amaltaro) [reply deploy/deploy-wmagent.sh:264]: Yes, my comment above isn't very clear. What I meant is that I need to remove this line once my deployment PR 1088 gets merged.

**Comment by amaltaro** [comment]
test this please

**Comment by amaltaro** [comment]
New issue created to follow up on these unit test failures, unrelated to these changes: https://github.com/dmwm/WMCore/issues/11194

**Pull request merged**
